packages/mdc-card/mdc-card.scss

```diff
@@ -48,7 +48,9 @@
 .mdc-card__media {
   @include mdc-card-media-aspect-ratio-base_;
 
+  display: flex;
```

Comment:
I'm not sure this makes much sense here, but also it seems like it would make more sense (and also work) if we added display / flex-direction to `.mdc-card__primary-action` instead.

In fact the 3 styles (not counting mixins) currently applied to `mdc-card` should maybe just be put in a selector that applies to both `mdc-card` and `mdc-card__primary-action`?